cmd/ucp_client_bundles.go

```diff
@@ -0,0 +1,148 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/thebsdbox/diver/pkg/ucp"
+)
+
+var username, bundleID, labelName string
+
+func init() {
+
+	// The username flag is required right now until I can find a nicer way of getting the currently logged-in
```

Comment:
https://github.com/thebsdbox/diver/blob/master/pkg/ucp/ucpAuth.go#L17 <- will return the username from the token.

Reply:
Thanks, anyway to do it without another round-trip to UCP? What about writing the username out to the token file on login so it's available on the Client?